src/components/CSecondaryNavigation.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="c-secondary-navigation">
+    <v-navigation-drawer
+      v-if="slots['list-items']"
+      :color="sideNavigationColor"
+      :order="navigationDrawerOrder"
+      :class="navigationDrawerClass"
+      elevation="0"
+      class="border-none"
+    >
+      <v-list color="primary" bg-color="transparent" variant="text">
+        <slot name="list-items"></slot>
+      </v-list>
+    </v-navigation-drawer>
+
+    <v-scale-transition v-if="slots['tab-items']">
+      <v-app-bar
+        v-if="mdAndDown"
+        :color="tabNavigationColor"
+        :order="appBarOrder"
+        height="48"
+      >
+        <c-primary-tabs>
+          <slot name="tab-items"></slot>
+        </c-primary-tabs>
```

Comment:
Would it be realistic to build the tab items and list items based of a common property? If so we could simplify the usage of the component and choose the correct version based on the breakpoint here.

Reply:
I thought about it, but I think, it will be easier like this. Think of Vuetify, if you build a list, you don't have an array you provide, but also use slots. I think, it is easier because you have every possibility to use the items as you like. Especially when it also comes to events and other properties.

Furthermore, we already have in our simple case the problem, that the items in the side navigation are not exactly the same as in the tabs navigation. The sidenav can have dividers, subtitles etc. the tabs don't